bedrock/firefox/tests/test_base.py

```diff
@@ -787,6 +787,64 @@ def test_fx_109_0_0_feature_variation_en(self, render_mock):
 
     # end 109.0 whatsnew tests
 
+    # begin 110.0 whatsnew tests
+
+    @override_settings(DEV=True)
+    def test_fx_110_0_0_de(self, render_mock):
+        """Should use whatsnew-fx110-de-v1 template for de locale"""
+        req = self.rf.get("/firefox/whatsnew/")
+        req.locale = "de"
+        self.view(req, version="110.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/whatsnew-fx110-de-v1.html"]
+
+    @override_settings(DEV=True)
+    def test_fx_110_0_0_de_v1(self, render_mock):
+        """Should use whatsnew-fx110-de-v1 template for de locale"""
+        req = self.rf.get("/firefox/whatsnew/?v=1")
+        req.locale = "de"
+        self.view(req, version="110.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/whatsnew-fx110-de-v1.html"]
+
+    @override_settings(DEV=True)
+    def test_fx_110_0_0_de_v2(self, render_mock):
+        """Should use whatsnew-fx110-de-v2 template for de locale"""
+        req = self.rf.get("/firefox/whatsnew/?v=2")
+        req.locale = "de"
+        self.view(req, version="110.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/whatsnew-fx110-de-v2.html"]
+
+    @override_settings(DEV=True)
+    def test_fx_110_0_0_fr(self, render_mock):
+        """Should use whatsnew-fx110-fr template for fr locale"""
+        req = self.rf.get("/firefox/whatsnew/?v=2")
```

Comment:
Probably don't need the `?v=2` in the URL here, I suspect that was copypasta from the test above.